### ISSUE TITLE
Make UCC more usable

### ISSUE THREAD

**Issue by mcneb10** [OldUnreal/UnrealTournamentPatches#1337]
Right now there are two issues with UCC (More specifically the Make commandlet) which bug me

1. No Linux/MacOS Support
- I'm working on a mutator that calls stuff in C++ and I want it to work on Windows, Linux, and MacOS
- I wrote a Makefile that calls UCC make and supports all three, but UCC make doesn't work on Linux/MacOS
2. It forces all projects to be in or have a symlink in the UnrealTournament folder
- This is very annoying. It allows no flexibility on where the projects can be. You get `Can't find files matching ..\project\Classes\*.uc` if the directory isn't symlinked or in the UT root directory. I'm prettysure this could be easily fixed by making UCC look its stuff in the current directory. Perhaps with a command line option enabiling that behavior for backwards compatibility

**Comment by SeriousBarbie** [comment]
Good idea! Currently I have my repository on a network drive and for compiling I mirror the project specific sub directory to the SYSTEM folder. 🙅

**Comment by mcneb10** [comment]
Is there a place where I could find up to date documentaion on UCC? I didn't see any of these options when I checked the OldUnreal wiki.

**Comment by SeriousBuggie** [comment]
Basic help is run in command line:
`ucc help`
`ucc help make`
Some switch is general and can be used with ucc or game itself.

https://github.com/OldUnreal/UnrealTournamentPatches/blob/929b0874407b23cc70aed68b586d89214cfdcc83/ReleaseNotes.md
> Added the -SETHOMEDIR= command line option. Normally, the game looks for the UnrealTournament.ini and User.ini files in \~/.utpg/System (on Linux), \~/Library/Application Support/Unreal Tournament/System (on Mac), or in UnrealTournament\System (on Windows). With this option, you can override the preferences path (e.g., ./ut-bin-x86 -sethomedir=\~/.loki/ut/System).

INI and USERINI it is well-know options for all Unreal based games.
For example: http://www.hypercoop.tk/infobase/archive/unrealtech/UTConsole.htm

**Comment by mcneb10** [comment]
Does sethomedir work on windows?

**Comment by mcneb10** [comment]
I wasn't able to get it to work. For now I just made my makefile create a symlink to the UT99 directory

**Comment by stijn-volckaert** [comment]
Could you try again with the 469d release candidate? We pushed some fixes for -sethomedir quite recently
